### ISSUE TITLE
Pam's HarvestCraft overrides grass drops of datura

### ISSUE THREAD

**Issue by klikli-dev** [klikli-dev/occultism#37]
**Describe the bug**
See title. Pam's seems to replace the entire loot list, instead of adding to it (Much like occultism did in the first public release)

**Expected behavior**
Datura seeds should drop even with Pam's installed. 

**Solution**
Contact Pam if they are interested in using the same solution that was contributed to Occultism.

**Comment by demfels** [comment]
So I don't have Pam's installed but also am not getting seeds... perhaps a conflict with a different mod?

**Comment by Shados** [comment]
I'm not seeing any Occultism seed drops from grass, but I am seeing seed drops from *multiple* other mods, so I don't think it is caused by some other mod replacing the loot table entirely...

**Comment by Shados** [comment]
So I just tested with *only* Occultism, and still got no datura seed drops. 

I then took a look into how the various mods in my pack are adding seed drops to grass:
- A couple are using [Global Loot Modifiers](https://mcforge.readthedocs.io/en/latest/items/globallootmodifiers/).
- One is just hooking the `onGrassBroken` event directly.
- I'm not using Pam's, but I looked anyway, and the 1.15 version appears to be using the same approach you are. I haven't checked to see if Pam's drops work.
